exercises/two-fer/README.md

```diff
@@ -0,0 +1,38 @@
+# Two Fer
+
+`Two-fer` or `2-fer` is short for two for one. One for you and one for me.
+
+Given a name, return a string with the message:
+
+```text
+One for name, one for me.
+```
+
+Where "name" is the given name.
+
+However, if the name is missing, return the string:
+
+```text
+One for you, one for me.
+```
+
+Here are some examples:
+
+|Name    |String to return
+|:-------|:------------------
+|Alice   |One for Alice, one for me.
+|Bob     |One for Bob, one for me.
+|        |One for you, one for me.
+|Zaphod  |One for Zaphod, one for me.
+
+### Submitting Exercises
+
+Note that, when trying to submit an exercise, make sure you're exercise file you're submitting is in the `exercism/vbnet/<exerciseName>` directory.
+
+For example, if you're submitting `bob.vb` for the Bob exercise, the submit command would be something like `exercism submit <path_to_exercism_dir>/vbnet/bob/bob.vb`.
```

Comment:
Using bob as the example is not necessary, as there are placeholders that you can use in the template to use the actual exercise's name: https://github.com/exercism/configlet#specname See the C# template for an example: https://github.com/exercism/csharp/blob/master/config/exercise_readme.go.tmpl

Reply:
Yeah, I was wondering about this. I'll get that changed.